### PR TITLE
Bump dependency versions

### PR DESCRIPTION
**Pull request by warrenvw**
Use latest versions of the `rabbitmq-ha` and `mongodb-replicaset` charts.

### CODE REVIEW

**Comment by warrenvw** [comment]
I've bumped all other dependencies, updated chart version and added an entry in the changelog.

**Comment by armab** [comment]
Per previous discussion, if new rabbitmq-ha Helm chart includes pod naming changes, then it needs more work TBD to verify if ST2 HA Helm templating works with those upstream changes well.

File: Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 # Update StackStorm version here to rely on other Docker images tags
 appVersion: 3.0dev
 name: stackstorm-ha
-version: 0.9.0
+version: 0.10.0
```
Comment (armab) [inline Chart.yaml]: Ideally version needs to be bumped just before the release, as by the fact this is not yet `0.10.0`.

**Comment by warrenvw** [comment]
Note that rabbitmq-ha `1.21.0` has a problem with the security settings / permissions, so I held its version at `1.20.1`.

File: Chart.yaml (hunk 1 above)
Comment (warrenvw) [reply Chart.yaml]: Oops, this slipped in there.. agreed.

**Comment by armab** [comment]
Useful info, It was a good opportunity to include that in code comments.

**Pull request merged**